canonical-web-and-design/jaas.ai
# Update and Switch to github actions

## Done

Testing sprint - Add full set of github actions

## QA

- Check that Github actions run correctly
- Wait for all checks to turn green
